src/code/views/simulation-run-panel-view.tsx

```diff
@@ -44,7 +44,8 @@ export class SimulationRunPanelView extends Mixer<SimulationRunPanelViewProps, S
 
   private renderToggleButton() {
     const iconClass = this.state.simulationPanelExpanded ? "inspectorArrow-collapse" : "inspectorArrow-expand";
-    const doNotRotate = getDefaultLang() === "ko";
+    const defaultLang = getDefaultLang();
+    const doNotRotate = (defaultLang === "ko") || (defaultLang === "pl");
```

Comment:
Probably should add Greek to this list as well.
<img width="221" alt="image" src="https://github.com/concord-consortium/building-models/assets/7716653/07bcd22a-b3b3-4593-9c94-9dc24e83e4ef">
German is pushing it but not as bad.
<img width="121" alt="image" src="https://github.com/concord-consortium/building-models/assets/7716653/16cb98b8-ae92-414e-aefa-22b4a3cafe89">